misc/logs/gen-user-data

```diff
@@ -21,7 +21,7 @@ my %cpu_lualibs = (
 );
 
 my @clibs = qw(
-    sregex resty-cli
+    sregex resty-cli lua-resty-signal lua-resty-shell
```

Comment:
This is not right. They are not pure C libraries. They are Lua libraries. The `lua-resty-signal` and `lua-tablepool` should goto `%cpu_lualibs` instead while `lua-resty-shell` should goto `@lualibs` instead.

Reply:
Ok, since both have C components I assume they'd go here for the `tv` mode, but will move them.

Reply:
Updated.